pkg/pipelines/bootstrap.go

```diff
@@ -73,12 +87,35 @@ func Bootstrap(quayUsername, baseRepo, prefix string) error {
 		}
 		fmt.Printf("%s---\n", data)
 	}
-
 	return nil
 }
 
 func pathToDownloadedFile(fname string) (string, error) {
-	return homedir.Expand(path.Join("~/Downloads/", fname))
+	return homedir.Expand(path.Join("~", fname))
+}
+
+//to get the github token file name
+func getTokenFileName(o *BootstrapOptions) (string, error) {
+	if o.TokenFileName == DefaultTokenFileName {
```

Comment:
This is wrong.

See my comment above, I think it's fine to use `token.txt` as long as we convert.

But this is confusing for users, we show them one file, but then look up another.

I think you can just use `homedir.Expand(o.TokenFilename)`, and in the main bootstrap flow.

Reply:
~ is not being recognized and due to that I was getting file not found. ~ need to be expanded.